Python/bytecodes.c

```diff
@@ -3260,7 +3262,7 @@ dummy_func(
             DISPATCH_INLINED(new_frame);
         }
 
-        inst(CALL_TYPE_1, (unused/1, unused/2, callable, null, args[oparg] -- res)) {
+        inst(CALL_TYPE_1, (unused/1, unused/2, callable, null, args[1] -- res)) {
```

Comment:
Why not go a step further and write `arg0` instead of `args[1]`? Then get rid of `obj` and just use `arg0` instead.

Reply:
Done